pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go

```diff
@@ -158,3 +160,25 @@ func TestGenDefaultSearchQueryUser(t *testing.T) {
 
 	assert.Equal(t, expected, actual)
 }
+
+func TestCheckOwnerTypeValid(t *testing.T) {
+	validOptions := []string{"org", "user"}
+
+	for _, option := range validOptions {
+		valid, err := checkOwnerTypeValid(option)
+
+		assert.True(t, valid)
+		assert.Nil(t, err)
+	}
+}
+
+func TestCheckOwnerTypeValidRandom(t *testing.T) {
```

Comment:
Do we need go fake it here? what if the word it generates is `user` or `org`. It would fail then.

Reply:
That's a good point... I can just use some hardcoded random words instead. I'll put in a new PR for that fix.

Reply:
Okay, the latest commit changes this out for some psuedo-random strings that I think are good test cases:

- `orgs`
- `sorg`
- `users`
- `suser`
- `random`
- `text`